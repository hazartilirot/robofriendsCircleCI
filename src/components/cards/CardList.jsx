import Card from './Card';

const CardList = ({ robots }) => {
  return (
    <div>
      {robots.map(robot => <Card key={robot.id} { ...robot } />)}
    </div>
  );
};
export default CardList;