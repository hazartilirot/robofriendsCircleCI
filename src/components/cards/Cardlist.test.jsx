import React from 'react';
import { shallow } from 'enzyme';
import CardList from './CardList';

it('expects a list of robots to render', () => {
  const robots = [
    {
      id: 1,
      name: 'John Doe',
      email: 'johndoe@exaple.com'
    },
  ];
  const wrapper = shallow(<CardList robots={robots} />)
  expect(wrapper).toMatchSnapshot();
})