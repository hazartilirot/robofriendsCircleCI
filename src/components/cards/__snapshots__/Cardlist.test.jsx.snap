// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects a list of robots to render 1`] = `ShallowWrapper {}`;
